internal/image/scan.go

```diff
@@ -22,33 +29,100 @@ func ScanImage(r reporter.Reporter, imagePath string) (ScanResults, error) {
 
 	allFiles := img.LastLayer().AllFiles()
 
-	scannedLockfiles := ScanResults{
+	scanResults := ScanResults{
 		ImagePath: imagePath,
 	}
+
+	inventories := []*extractor.Inventory{}
+
 	for _, file := range allFiles {
 		if file.fileType != RegularFile {
 			continue
 		}
-		parsedLockfile, err := extractArtifactDeps(file.virtualPath, img.LastLayer())
+
+		// TODO: Currently osv-scalibr does not correctly annotate OS packages
+		// causing artifact extractors to double extract elements here.
+		// So let's skip all these directories for now.
+		// See (b/364536788)
+		if strings.HasPrefix(file.virtualPath, "/usr/local/") {
```

Comment:
why skip scanning if  /usr/local/ ? 

/usr/local generally has stuff not installed by OS package managers right? it's everything else under /usr that's problematic wrt double scanning?

Reply:
I think we found a lot of OS packages get installed here. @hogo6002 IIRC you found a lot of go compiler binaries here?

Reply:
Yes, we found a lot of go compiler binaries there

Reply:
Example: 
`GO-2023-2375` has been found in the actual project binary.
![image](https://github.com/user-attachments/assets/329265a1-729f-40e0-8caf-f8ceafd06bc3)
But it also shows 20 times in /usr/lib
![image](https://github.com/user-attachments/assets/21cf8808-b331-40a1-a919-5b02ecc57d02)
![image](https://github.com/user-attachments/assets/06de052a-7d95-4612-bf29-fa20cbc870e4)

Reply:
Ah gotcha, I'll update this path. I think I found some npm packages in `/usr/local` though. I'll double check

Reply:
Checking https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

It looks like we should ignore the entire /usr directory. I think everything in there is written by os package managers, since it is suppose to be read only.